tools/metadata.json

```diff
@@ -3,5 +3,6 @@
   "PreviewReleaseTag": "v7.0.0-rc.2",
```

Comment:
@adityapatwardhan, your last commit had 1 failures in `PowerShell-CI-macos`
Verify sub-directory creation under root.Can create a sub directory under root path
```powershell
Expected path '\TestDir-19d3f162-077a-48ed-942c-8a9baf03a45d' to exist, but it did not exist.
at <ScriptBlock>, /Users/runner/runners/2.164.8/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/FileSystem.Tests.ps1: line 1467
1467:         $dirPath | Should -Exist
```